components/FormContainer.vue

```diff
@@ -0,0 +1,271 @@
+									<button @click="nextStep()">Start</button>
+									<span class="text-xs">press <strong>Enter</strong> ↵</span>
+									<p class="text-xs mt-2">🕒 Takes 4 minutes</p>
+								</div>
+							</div>
+
+							<button v-if="hasPreviousState" @click="previousStep">Previous</button>
+							<button v-if="hasNextState && stateIndex > 0" @click="nextStep">
+								Next
+							</button>
+							<button v-if="!hasNextState" @click="submitForm">Submit</button>
+						</div>
+					</div>
+				</Transition>
+			</div>
+			<div class="flex flex-wrap p-4">
+				<ProgressBar :percentage="percentage" class="mx-2 mb-2 " />
+			</div>
+		</article>
+	</div>
+</template>
+<script>
+import { mapState } from "vuex";
+
+import FormIntro from "./FormIntro.vue";
+import FormSingleField from "./FormSingleField.vue"
+import FormMultipleSelect from "./FormMultipleSelect.vue"
+import FormMultipleInput from "./FormMultipleInput.vue"
+
+import ProgressBar from "./ProgressBar.vue";
+
+export default {
+	mounted: function () {
+		window.addEventListener('keyup', (e) => { this.keyboardControl(e) })
+	},
+	data: () => {
+		return {
+			stateIndex: 0,
+			submitting: false,
+			formData: {
+				username: "",
+				learnings: [],
+				sharings: [],
+				newLearnings: [],
+				newSharings: [],
+				comment: ''
+			},
+		}
+	},
+	computed: {
+		...mapState(['supabase']),
+		currentLearning: function () {
+			// Get topics that have a value in the learning mapping 
+			// const learningTopics = this.mapping.filter(item => item.learner).map(item => item.topic)
+			// Filter topics based on this
+			// const topicsWithLearners = this.currentTopics.filter(item => learningTopics.includes(item.id))
+			return this.$store.state.supabase.topics
+		},
+		currentSharing: function () {
+			// Get topics that have a value in the sharing mapping 
+			// const sharingTopics = this.mapping.filter(item => item.sharer).map(item => item.topic)
+			// Filter topics based on this
+			// const topicsWithSharers = this.currentTopics.filter(item => sharingTopics.includes(item.id))
+			return this.$store.state.supabase.topics
+		},
+		states() {
+			return [
+				{
+					component: FormIntro,
+					introduction: 'Hi! 👋 This is an invitation to connect to other MozFest participants via skills, interests and fascinations.',
+					followup: 'Your input will be used to connect you with other participants on Slack and will be shown on the MozFest Gradual page.',
+					updateFunction: () => { }
+				},
+				{
+					component: FormSingleField,
+					introduction: "What's your @UserName on the MozFest Slack?",
+					followup: "Please write down your exact user name, including the @ sign.",
+					value: this.formData.username,
+					updateFunction: (value) => {
+						this.setSingleField('username', value)
+					}
+				},
+				{
+					component: FormMultipleSelect,
+					introduction: "These are some topics suggested by other Mozfest participants! Anything here you'd like to learn more about?",
```

Comment:
this and the next question look very much alike. our testing shows that people get confused by this. For this reason in the typeform version the phrase "you'd like to learn" is displayed in bold. Could we somehow facilitate this? 

We could consider loading states from markdown files using nuxt content, but that's probably a larger upgrade for later. Could we do something intermediate, e.g. using the `v-html` directive?

Reply:
Updated this component to v-html for now. If I've time I'll circle back and see if I can use Vue content as we discussed as the ideal solution.

Reply:
I've had a look into this and the added complexity is that the components also contain JS logic. 

I think what we want is probably possible but it's a bigger task than we previously thought.